CHANGELOG.md

```diff
@@ -3,18 +3,24 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+### Changed
+- Removed bower_components as a dev dependency
```

Comment:
This is not a change in the API/functionality. It should probably go under ###Support